tests/zero-downtime-scaling/results/1.2.0/1.2.0.md

```diff
@@ -0,0 +1,578 @@
+10.150.0.26 - - [20/Mar/2024:16:26:29 +0000] "GET /coffee HTTP/1.1" 499 0 "-" "-"
+10.150.0.27 - - [20/Mar/2024:16:26:31 +0000] "GET /tea HTTP/1.1" 499 0 "-" "-"
+```
+
+
+### Scale up abruptly
+
+HTTP wrk output
+
+```text
+ wrk -t2 -c2 -d2m --latency --timeout 2s  http://cafe.example.com/coffee
+Running 2m test @ http://cafe.example.com/coffee
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.59ms  827.86us  24.57ms   92.30%
+    Req/Sec   651.66    109.00     0.97k    71.96%
+  Latency Distribution
+     50%    1.45ms
+     75%    1.76ms
+     90%    2.17ms
+     99%    4.68ms
+  155782 requests in 2.00m, 55.53MB read
+Requests/sec:   1297.24
+Transfer/sec:    473.54KB
+```
+
+HTTP Graph
+
+![abrupt-up-http.png](25-node/abrupt-up-http.png)
+
+HTTPS wrk output
+
+```text
+wrk -t2 -c2 -d2m --latency --timeout 2s  https://cafe.example.com/tea
+Running 2m test @ https://cafe.example.com/tea
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     3.86ms   39.67ms   1.06s    99.59%
+    Req/Sec   640.00    110.00     0.94k    71.55%
+  Latency Distribution
+     50%    1.47ms
+     75%    1.79ms
+     90%    2.23ms
+     99%    6.24ms
+  152404 requests in 2.00m, 53.25MB read
+Requests/sec:   1269.04
+Transfer/sec:    454.07KB
+```
+
+HTTP Graph
+
+![abrupt-up-https.png](25-node/abrupt-up-https.png)
+
+Logs
+
+- NGF error logs - none
+- NGINX error/warn logs - none
+- NGINX Access Logs:
+  - 309K 200s
+
+
+### Scale down abruptly
+
+HTTP wrk output
+
+```text
+wrk -t2 -c2 -d2m --latency --timeout 2s  http://cafe.example.com/coffee
+Running 2m test @ http://cafe.example.com/coffee
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.63ms    0.94ms  37.11ms   93.21%
+    Req/Sec   634.30    100.40     0.96k    78.64%
+  Latency Distribution
+     50%    1.47ms
+     75%    1.81ms
+     90%    2.27ms
+     99%    4.74ms
+  126662 requests in 2.00m, 45.15MB read
+  Socket errors: connect 0, read 0, write 0, timeout 2
+Requests/sec:   1055.32
+Transfer/sec:    385.23KB
+```
+
+
+HTTP Graph
+
+![abrupt-down-http.png](25-node/abrupt-down-http.png)
+
+HTTPS wrk output
+
+```text
+wrk -t2 -c2 -d2m --latency --timeout 2s  https://cafe.example.com/tea
+Running 2m test @ https://cafe.example.com/tea
+  2 threads and 2 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency     1.70ms    0.97ms  36.95ms   93.59%
+    Req/Sec   603.54     96.80     0.90k    77.44%
+  Latency Distribution
+     50%    1.53ms
+     75%    1.89ms
+     90%    2.37ms
+     99%    4.74ms
+  118369 requests in 2.00m, 41.36MB read
+  Socket errors: connect 0, read 2, write 0, timeout 2
+Requests/sec:    986.21
+Transfer/sec:    352.87KB
+```
+
+
+HTTPS Graph
+
+![abrupt-down-https.png](25-node/abrupt-down-https.png)
+
+Logs
+
+- NGF error logs - 25
```

Comment:
what about NGF errors here?

Reply:
re-checked and it was the same error related to telemetry. Added it to the document